js/domUpdates.js

```diff
@@ -1,59 +1,99 @@
-const players = [];
-const clues = [];
-const rounds = [];
-let game;
+const jeopardy = {
+  players: [],
+  clues: [],
+  rounds: [],
+  game: {},
+  categoryNames: {
+    unitedStatesHistory: 'United States History',
+    lifeSciences: 'Life Sciences',
+    publicHealth: 'Public Health',
+    educationJargon: 'Education Jargon',
+    nameThatBoardGame: 'Name That Board Game',
+    americanLiterature: 'American Literature',
+    biographies: 'Biographies',
+    americanCities: 'American Cities',
+    food: 'Food',
+    cableTV: 'Cable TV'
+  }
+};
```

Comment:
Realistically, hard-coding an entire new dataset that looks how you want isn't the way to go about this. Imagine if you application's dataset had 10,000 categories. It would be kind of a pain to hardcode all of that information into a nicer format, and also cause a huge performance hit. Now imagine if the number of categories kept increasing -- you'd have to make sure you keep this hard-coded dataset up-to-date with the latest data. You won't see this done in the real-world because it's unmaintainable and unsustainable. A better option here would be to use prototype methods to massage the original data into a format you want, only where you need it. e.g. if you're trying to display nicely formatted category names on the DOM, in your method for displaying these to the DOM you should do a `map` over the original dataset to convert the camelCase to a space (you'd need to write some logic to find the first capital letter in the string and put a space before it).